umake/frameworks/baseinstaller.py

```diff
@@ -396,7 +398,8 @@ def decompress_and_install_done(self, result):
             UI.return_main_screen(status_code=1)
 
         self.post_install()
-
+        if self.desktop_filename:
```

Comment:
Maybe, we can even set:
if self.exec_link_name (to abstract from desktop_filename)